### DISCUSSION TITLE
implementing Limelight 3a into localizer

### DISCUSSION THREAD

**Discussion by Ls729-h** [acmerobotics/road-runner-quickstart#411] [Q&A]
Hi, @rbrott 
we are currently trying to integrate the newly released limelight 3a and an extended kalman filter into the localizer.
where should i put `limelight.start()` and `limelight.getLatestResult()`? Which function in the mecanum drive class is periodic?

thx

**Comment by rbrott** [comment]
This is a good periodic function in `MecanumDrive`: https://github.com/acmerobotics/road-runner-quickstart/blob/5f35f4c22c1ae7c0be5b35da0961c8f3a181ad31/TeamCode/src/main/java/org/firstinspires/ftc/teamcode/MecanumDrive.java#L443. I would assume you can put `start()` in the constructor, though I'm not familiar with the limelight API.
